Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -37,14 +37,17 @@ check_ldd_version() {
   local ldd_version_major
   local ldd_version_minor
 
+  local minimum_required_major=${HOMEBREW_MINIMUM_GLIBC_VERSION%.*}
+  local minimum_required_minor=${HOMEBREW_MINIMUM_GLIBC_VERSION#*.}
+
   if [[ $(/usr/bin/ldd --version) =~ \ [0-9]\.[0-9]+ ]]
   then
     ldd_version=${BASH_REMATCH[0]// /}
     ldd_version_major=${ldd_version%.*}
     ldd_version_minor=${ldd_version#*.}
-    if (( ldd_version_major < 2 || ldd_version_minor < 13 ))
+    if (( ldd_version_major < minimum_required_major || ldd_version_minor < minimum_required_minor ))
     then
-      odie "Vendored tools require system Glibc 2.13 or later."
+      odie "Vendored tools require system Glibc $HOMEBREW_MINIMUM_GLIBC_VERSION or later (yours is $ldd_version)."
```

Comment:
Would be good to get this function no-oping either when `HOMEBREW_LINUX` isn't set and perhaps when `HOMEBREW_MINIMUM_GLIBC_VERSION` isn't set to ensure it doesn't fail when called on macOS. Might also be worth renaming to `check_glibc_ldd_version` or `check_linux_glibc_ldd_version` or something